uPortal-webapp/src/main/webapp/WEB-INF/flows/edit-portlet/portletConfig.jsp

```diff
@@ -464,13 +464,19 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
                                                 <td class="col-sm-3 text-nowrap">
                                                     <spring:message code="edit.browse"/>
                                                     <a href="javascript:void(0)" data-toggle="tooltip" data-placement="top" title="<spring:message code='edit.browse.tooltip'/>">
-                                                        <i class="fa fa-info-circle"></i>
+                                                        <i class="fa fa-info-circle" aria-hidden="true"></i> <span class="sr-only"><spring:message code='edit.browse.tooltip'/></span>
                                                     </a>
                                                 </td>
                                                 <td class="col-sm-3 text-nowrap">
                                                     <spring:message code="edit.subscribe"/>
                                                     <a href="javascript:void(0)" data-toggle="tooltip" data-placement="top" title="<spring:message code='edit.subscribe.tooltip'/>">
-                                                        <i class="fa fa-info-circle"></i>
+                                                        <i class="fa fa-info-circle" aria-hidden="true"></i> <span class="sr-only"><spring:message code='edit.subscribe.tooltip'/></span>
+                                                     </a>
+                                                </td>
+                                                <td class="col-sm-3 text-nowrap">
```

Comment:
You just added a bonus `col-sm-3` to an existing HTML structure -- is that okay?  (IIRC Bootstrap columns are meant to add up to 12.)

Reply:
Check with @ChristianMurphy on this one.

Reply:
Bootstrap 3 grid is percentage based, so this is the equivalent of
``` css
@media (min-width: 768px) { /* smaller screen sizes will apply no width rule */
    .col-sm-4 {
        width: 33.33333333%;
    }
}
```

functionally it works.
Semantically, using `col-sm-3` here is a bit wonky.